test/pools/integration/Cases.t.sol

```diff
@@ -11,6 +11,7 @@ contract TestCases is BaseTest {
     /// forge-config: default.isolate = true
     function testPoolCreation() public returns (PoolId poolId, ShareClassId scId) {
         cv.registerAsset(USDC_C2, "USD Coin", "USDC", 6);
+        cv.registerAsset(EUR, "Euro", "EUR", 12);
```

Comment:
Should not this be a non-national currency? I would use `USDT_C2` or so, to indicate:
1. It's a non-national currency.
2. It's created from chain 2.